Sources/DatadogObjc/RUM+objc.swift

```diff
@@ -0,0 +1,219 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import UIKit
+import Datadog
+
+internal struct PredicateBridge: UIKitRUMViewsPredicate {
+    let objcPredicate: DDUIKitRUMViewsPredicate
+    func rumView(for viewController: UIViewController) -> RUMView? {
+        return objcPredicate.rumView(for: viewController)?.swiftView
+    }
+}
+
+@objcMembers
+public class DDRUMView: NSObject {
+    let swiftView: RUMView
+
+    public var path: String { swiftView.path }
+    public var attributes: [String: Any] { swiftView.attributes }
+
+    /// Initializes the RUM View description.
+    /// - Parameters:
+    ///   - path: the RUM View path, appearing as "PATH" in RUM Explorer.
+    ///   - attributes: additional attributes to associate with the RUM View.
+    public init(path: String, attributes: [String: Any]) {
+        swiftView = RUMView(
+            path: path,
+            attributes: castAttributesToSwift(attributes)
+        )
+    }
+}
+
+@objc
+public protocol DDUIKitRUMViewsPredicate {
+    /// The predicate deciding if the RUM View should be started or ended for given instance of the `UIViewController`.
+    /// - Parameter viewController: an instance of the view controller noticed by the SDK.
+    /// - Returns: RUM View parameters if received view controller should start/end the RUM View, `nil` otherwise.
+    func rumView(for viewController: UIViewController) -> DDRUMView?
+}
+
+@objc
+public enum DDRUMErrorSource: Int {
+    /// Error originated in the source code.
+    case source
+    /// Error originated in the network layer.
+    case network
+    /// Error originated in a webview.
+    case webview
+    /// Custom error source.
+    case custom
+
+    fileprivate var swiftType: RUMErrorSource {
+        switch self {
+        case .source: return .source
+        case .network: return .network
+        case .webview: return .webview
+        case .custom: return .custom
+        default: return .custom
+        }
+    }
+}
+
+@objc
+public enum DDRUMUserActionType: Int {
+    case tap
+    case scroll
+    case swipe
+    case custom
+
+    fileprivate var swiftType: RUMUserActionType {
+        switch self {
+        case .tap: return .tap
+        case .scroll: return .scroll
+        case .swipe: return .swipe
+        case .custom: return .custom
+        default: return .custom
+        }
+    }
+}
+
+@objcMembers
+public class DDRUM: NSObject {
+    let sdkRUM: DDRUMMonitor
```

Comment:
We can keep uniformed naming in obj-c public API: `DDRUMMonitor` if we namespace the Swift one:
```swift
public class DDRUMMonitor: NSObject {
   let sdkRUMMonitor: Datadog.DDRUMMonitor
}
```

Reply:
there already is a `DDRUMMonitor` in Swift target, i used `DDRUM` to avoid confusion